get_rabbit_definitions.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+# assumes the container to be running already
+
+docker exec -it gamification_rabbitmq_1 rabbitmqadmin -q export rabbit.definitions.json
+docker exec -it gamification_rabbitmq_1 cat rabbit.definitions.json
```

Comment:
Could you add a more detailed description to this script? Why would I want to execute it? What exactly are these definitions?